doc/README.hi.md

```diff
@@ -0,0 +1,143 @@
+
+इस दौरान अगर आप स्रोत बदलना चाहते हैं (मान लीजिए NewPipe का मूल हिस्सा टूट गया है और F-Droid पर अब भी नवीनतम संस्करण नहीं है), हम इस तरीके का इस्तेमाल करने की सलाह देंगे:
+1. सेटिंग्स > सामग्री > डेटाबेस निर्यात करें के ज़रिए अपने डेटा को बैकअप कर लें ताकि आपके पास अपने देखे गए वीडियो, सदस्यताएँ और प्लेलिस्ट्स हो
+2. NewPipe को अस्थापित करें
+3. नए स्रोत से APK डाउनलोड करें और उसे स्थापित करें
+4. सेटिंग्स > सामग्री > डेटाबेस आयात करें के ज़रिए पहले चरण के डेटा को आयात करें
+
+## योगदान करें
+चाहे आपके विचार जोड़ना चाहे, या अनुवाद, डिज़ाइन में बदलाव, कोड में सफ़ाई, या कोड में भारी बदलाव, सहायता ज़रूर करें।
+जितने बदलाव हो यह उतना ही बेहतर होता जाता है!
+
+अगर आप शामिल होना चाहते हैं, हमारे [योगदान की टिप्पणियाँ](.github/CONTRIBUTING.md) देखें।
+
+<a href="https://hosted.weblate.org/engage/newpipe/">
+<img src="https://hosted.weblate.org/widgets/newpipe/-/287x66-grey.png" alt="अनुवाद की स्थिति" />
+</a>
+
+## दान करें
+अगर आपको NewPipe पसंद है, एक छोटे-से दान से हम खुश होंगे। आप या तो बिटकॉइन भेज सकते हैं या फिर Bountysource या Liberapay से दान कर सकते हैं। NewPipe को दान करने के बारे में अधिक जानकारी के लिए कृपया हमारा [वेबसाइट](https://newpipe.net/donate) देखें।
+
+<table>
+  <tr>
+    <td><img src="https://bitcoin.org/img/icons/logotop.svg" alt="Bitcoin"></td>
+    <td><img src="../assets/bitcoin_qr_code.png" alt="बिटकॉइन पर दान करने के लिए QR कोड" width="100px"></td>
+    <td><samp>16A9J59ahMRqkLSZjhYj33n9j3fMztFxnh</samp></td>
+  </tr>
+  <tr>
+    <td><a href="https://liberapay.com/TeamNewPipe/"><img src="https://upload.wikimedia.org/wikipedia/commons/2/27/Liberapay_logo_v2_white-on-yellow.svg" alt="Liberapay" width="80px"></a></td>
+    <td><a href="https://liberapay.com/TeamNewPipe/"><img src="../assets/liberapay_qr_code.png" alt="liberapay.com पर NewPipe को देखें" width="100px"></a></td>
+    <td><a href="https://liberapay.com/TeamNewPipe/donate"><img src="../assets/liberapay_donate_button.svg" alt="Liberapay के ज़रिए दान करें" height="35px"></a></td>
+  </tr>
+  <tr>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/Bountysource.png/320px-Bountysource.png" alt="Bountysource" width="190px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="../assets/bountysource_qr_code.png" alt="bountysource.com पर NewPipe को देखें" width="100px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe/issues"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f" height="30px" alt="देखें कि आप कितने बाउंटी कमा सकते हैं।"></a></td>
+  </tr>
+</table>
+
+## निजता नीति

```

Comment:
Where did you get this translation from? Google uses "गोपनीयता और शर्तें" for its Hindi privacy policy.

Reply:
[MediaWiki](https://www.mediawiki.org) - I'm mostly a translator there so I have used some words that i do not see translations for here, from their vocabulary.

Reply:
So which one should we go with? I'm leaning towards Google's translation, simply because it would have been used across all Google services, making it more popular.